doc/cephfs/troubleshooting.rst

```diff
@@ -278,6 +278,17 @@ Identify the stuck commands and examine why they are stuck.
 Usually the last "event" will have been an attempt to gather locks, or sending
 the operation off to the MDS log. If it is waiting on the OSDs, fix them. 
 
+.. _
+
+   I have a couple of questions about this paragraph.
+
+   1. What is "it" in the sentence "If it is waiting on the OSDs, 
+      fix them."
+
+   2. What does "fix them" mean in the sentence "If it is waiting
+      on the OSDs, fix them."? Does this mean that the reader
+      should fix the OSDs? What does it mean to "fix" an OSD?
```

Comment:
"fix" is applicable only when the reason behind the stuck commands is identified. I am amazed how easily we confuse users :)

That said, I think this required a special section on how to identify the reason behind a stuck command and then document here how to unblock such commands.